torch/onnx/_internal/exporter/_building.py

```diff
@@ -431,17 +431,17 @@ def _process_python_sequences(
                 elif val is None:
                     # Skip None values
                     continue
-                elif isinstance(arg, (ir.Tensor, ir.TensorProtocol)):
+                elif isinstance(val, (ir.Tensor, ir.TensorProtocol)):
```

Comment:
@justinchuby Actually I am not sure about this one being a bug or not.

Reply:
It's a bug. Thanks for fixing!